package.json

```diff
@@ -55,6 +55,7 @@
     "istanbul-lib-report": "^3.0.0",
     "istanbul-reports": "^3.0.0",
     "jest": "workspace:packages/jest",
+    "jest-environment-jsdom-fifteen": "^1.0.2",
```

Comment:
?

Reply:
@SimenB When tests were run on `Node v10.x`, I've got the `ReferenceError: globalThis is not defined` error. This doesn't happen on the other versions of node. The following [solution](https://github.com/nrwl/nx/issues/3776#issuecomment-699013760) was implemented to fix the problem in this [commit](https://github.com/facebook/jest/pull/10564/commits/c9069fdb25b5afb6abf0747b7e991cb2100634d2).

Now, In a previous comment you said that `Node v10.x` is not supported by Jest anymore, so I could just remove it, but then the Node CI will throw errors.

Please let me know how do you want to proceed with this. 😄

Reply:
> @SimenB When tests were run on `Node v10.x`, I've got the `ReferenceError: globalThis is not defined` error. This doesn't happen on the other versions of node. The following [solution](https://github.com/nrwl/nx/issues/3776#issuecomment-699013760) was implemented to fix the problem in this [commit](https://github.com/facebook/jest/pull/10564/commits/c9069fdb25b5afb6abf0747b7e991cb2100634d2).

Right, https://github.com/jsdom/jsdom/issues/2961 - an old friend 😀 

> Now, In a previous comment you said that `Node v10.x` is not supported by Jest anymore, so I could just remove it, but then the Node CI will throw errors.

Node `<` 10 is not supported, 10 is. Sorry about the confusion

> Please let me know how do you want to proceed with this. 😄

Don't install `fifteen`. Not sure what test failed, but just use `jest-environment-node` instead instead of a `jsdom` one

Reply:
@SimenB 
> Node < 10 is not supported, 10 is. Sorry about the confusion

Oh yeah you're right, you said `doesn't support node<10`, my bad 😄 .

> Don't install fifteen. Not sure what test failed, but just use jest-environment-node instead instead of a jsdom one

Will do 👍